python/desc/sims/GCRCatSimInterface/HostImages.py

```diff
@@ -0,0 +1,100 @@
+import os
+import numpy as np
+import pandas as pd
+from lsst.sims.utils import angularSeparation
+
+__all__ = ['hostImage']
+
+class hostImage(object):
+
+    def __init__(self, ra_center, dec_center, fov):
+
+        self.ra = ra_center
+        self.dec = dec_center
+        self.radius = fov
+
+
+    def format_catalog(self, df_line, fits_file_name, image_dir):
+        """Formats the output instance catalog to include entries for the FITS stamps produced by generate_lensed_host_***.py
+		
+		Parameters: 
+		-----------
+		df_line: string
+			pandas data frame line
+		fits_file_name: string
+			the filename of the FITS stamp
+		image_dir: string 
+			the location of the FITS stamps 
+        	"""
+        fits_info = fits_file_name.split('_')
+        lens_id = np.int(fits_info[0])
+        sys_magNorm = np.float(fits_info[1])
+        gal_type = fits_info[2].split('.')[0]
+
+        galaxy_id = np.right_shift(lens_id, 10)
+        galaxy_id *= 10000
+        galaxy_id += 4*df_line['twinkles_system']
+        sys_id = np.left_shift(galaxy_id, 10)
+        if gal_type == 'bulge':
+            sys_id += 97
+        elif gal_type == 'disk':
+            sys_id += 107
+        else:
+            raise ValueError('Invalid Galaxy Component Type in filename')
+
+        cat_str = 'object %i %f %f %f %s %f 0 0 0 0 0 %s 0.01 0 CCM %f %f CCM %f %f\n' % (sys_id,
+                                                                      df_line['raPhoSim_lens'],
+                                                                      df_line['decPhoSim_lens'],
+                                                                      sys_magNorm,
+                                                                      df_line['sedFilepath'],
+                                                                      df_line['redshift'],
+                                                                      str(lens_id) + "_" + str(sys_magNorm) + "_" + str(gal_type) + ".fits",

```

Comment:
I'm wondering if this file specification for the FITS image files for the lensed object should also prepend the subfolder name, e.g., `agn_lensed_bulges`.  My understanding is that phosim expects a single `--imageDir` path for these sorts of files, so the instance catalog entry needs to be a correct relative path to that `imageDir` value.